Test/Unit/Helper/PaymentResponseHandlerTest.php

```diff
@@ -29,6 +28,10 @@
 use Magento\Sales\Model\OrderRepository;
 use Magento\Sales\Model\Order\Status\HistoryFactory;
 use Adyen\Payment\Helper\StateData;
+use Adyen\Payment\Model\ResourceModel\PaymentResponse\Collection;
+use Adyen\Payment\Model\ResourceModel\PaymentResponse\CollectionFactory;
+use Adyen\Payment\Helper\Config as Config;
```

Comment:
Import alias is redundant
```suggestion
use Adyen\Payment\Helper\Config;
```